src/test/java/com/glovoapp/planningpoker/e2e/spectatorbox/SpecSelectedTest.java

```diff
@@ -0,0 +1,47 @@
+package com.glovoapp.planningpoker.e2e.spectatorbox;
+
+import com.glovoapp.planningpoker.e2e.WebDriverFactory;
+import com.glovoapp.planningpoker.e2e.pages.MainPage;
+import com.glovoapp.planningpoker.e2e.pages.SessionPage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Optional;
+
+import static com.glovoapp.planningpoker.e2e.WebDriverFactory.getDriver;
+
+class SpecSelectedTest {
```

Comment:
Please do not use abbreviations in class names.

```suggestion
class SpectatorSelectionTest {
```